iam.py

```diff
@@ -375,7 +375,7 @@ def daily_update():
 
     eastern = timezone('US/Eastern')
     utc = timezone('UTC')
-    eastern_time = utc.localize(datetime.utcnow()).astimezone(eastern).hour
+    eastern_time = utc.localize(datetime.utcnow()).astimezone(eastern).time
     if eastern_time < dt_time(8,50) or eastern_time.time() > dt_time(9,10):
```

Comment:
something is not right here. does this need the .time() or no?

Reply:
It doesn't, but line 378 needed parentheses at the end. Should be fixed and actually working now